.gitignore

```diff
@@ -0,0 +1,53 @@
+HELP.md
```

Comment:
I guess this might be a historical relic, but what's wrong with having a file called `HELP.md`?